src/wmnds/components/content-tiles/small1/_small1.scss

```diff
@@ -0,0 +1,12 @@
+.wmnds-content-tile {
+  padding: $size-sm;
+  border-bottom: 5px solid get-color(primary);
+
+  &--yellow {
+    background-color: get-color(plannedDisruption);
+
+    .read-more {
```

Comment:
Make sure this is a BEM class.

https://github.com/wmcadigital/wmn-design-system/blob/master/doc/contributing/css.md